### PR TITLE
Ticket/1455 cthp card alignment

### LINKED ISSUE #1455: Alignment Issues on CTHP Guide Cards without PDQ Summary
When there is no PDQ summary, the description should still have the same alignment as the other cards with PDQ link headings 

image

### PR DESCRIPTION
**Pull request by arcepaul**
Closes #1455 

Added styling to paragraphs when they are the first paragraph in the card body so that the alignment is consistent when certain elements are absent from the content.

Also fixes display logic for pdq list heading.  PDQ link heading should only appear if provided AND links are present.

### CODE REVIEW

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cthp/paragraph--cgov-cthp-guide-card.html.twig [hunk 1]
```diff
@@ -6,7 +6,7 @@
       {{ content.field_cthp_guide_card_desc }}
     {% endif %}
 
-    {% if content.field_cthp_pdq_link_heading|field_value or content.field_cthp_pdq_links["#items"] %}
+    {% if content.field_cthp_pdq_link_heading|field_value and content.field_cthp_pdq_links["#items"] %}
       <h4 class="cthp-pdq-label">
```
Comment (BrendanBeltzNIH) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cthp/paragraph--cgov-cthp-guide-card.html.twig]: If both fields need the content.field_cthp_pdq_links["#items"] to be true, why not just wrap both in the one conditional and only the heading field in the other?

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss [hunk 2]
```diff
@@ -250,7 +255,7 @@
 
 .more-info-list ul li:before,
 .cthp-pdq-label ul li:before,
-.cardBody h4 + ul li:before,
+.cardBody ul li:before,
```
Comment (BrendanBeltzNIH) [inline docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss]: Can I poke my nose in and ask why we're making this markup change? These aren't the same selectors on prod, as demonstrated here: 
image

Comment (BrendanBeltzNIH) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss]: Oh, I just noticed you set me as a reviewer. So I guess I'm not technically sticking my nose in. :)

Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss]: Speaking with the IA, if for some strange reason a content editor added a list without a list heading and didn't realize how dumb it looks, the list SHOULD be bulleted.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cthp/paragraph--cgov-cthp-guide-card.html.twig (hunk 1 above)
Comment (arcepaul) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/templates/content/cthp/paragraph--cgov-cthp-guide-card.html.twig]: good point.  fixing now.

File: docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss (hunk 2 above)
Comment (BrendanBeltzNIH) [reply docroot/profiles/custom/cgov_site/themes/custom/cgov/cgov_common/src/entrypoints/cthp/CTHP.scss]: Stupid content editors...

**Comment by BrendanBeltzNIH** [comment]
I'm happy. Or as happy as can be expected in durpal.

**Pull request merged**